.travis.yml

```diff
@@ -24,7 +24,7 @@ script:
   - cd docs; make html
   # Run the test suit with coverage
   - cd ..
-  - travis_wait 30 coverage run --source=axelrod -m unittest discover
+  - travis_wait 60 coverage run --source=axelrod -m unittest discover
```

Comment:
Maybe we should work on decreasing the time it takes all the tests to run

Reply:
> Maybe we should work on decreasing the time it takes all the tests to run

That was mainly the purpose of the work I did on https://github.com/Axelrod-Python/Axelrod/pull/1121.

There I reduced the test runs times on travis - where it's very long - from approx 40 mins to approx 20 mins. (Note that locally the full test suite takes about 10 mins on my small laptop - similar on appveyor.)

Just now it failed on a timeout which would have just been a random event. The 30 minute limit was arbitrary there's no reason not to increase it to 60. It hadn't timed out for a while since #1121 (before that it was essentially timing out every other run) but we'll keep an eye on it.

Reply:
> That was mainly the purpose of the work I did on #1121.

(Just in case you only read the title of that issue, the rest of the comments there detail the work I went through minimising test suite run time and why I did it there. :+1:)